### DISCUSSION TITLE
Any plans for desktop Linux app.

### DISCUSSION THREAD

**Discussion by vishalduniya** [iSoron/uhabits#1211] [General Discussion]
I would love to see a desktop Linux habit tracking application. Till now, could not find one on Linux.

**Comment by hiqua** [comment]
It's planned: https://github.com/iSoron/uhabits/issues/486

But no ETA, don't count on it any time soon. Feel free to get involved to speed it up!

**Comment by 12people** [comment]
I posted this in a [different discussion](https://github.com/iSoron/uhabits/discussions/711), but it's just as relevant here:

There's a different free and open-source habit tracker that could be ported to Linux: [Habo](https://github.com/xpavle00/Habo), which is a GPL-licensed multi-platform Flutter app.

It's only on Android and iOS for now, but I'm working on a pull request to bring it to Linux too.
